package-lock.json

```diff
@@ -0,0 +1,1914 @@
+{
+  "name": "jsmgclient",
+  "version": "0.0.1",
+  "lockfileVersion": 2,
+  "requires": true,
+  "packages": {
+    "": {
+      "name": "jsmgclient",
+      "version": "0.0.1",
+      "license": "Apache-2.0",
```

Comment:
Since you are mentioning license here, I would also add  a LICENSE file(https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository)

Reply:
https://github.com/memgraph/jsmgclient/pull/9